app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
```

Comment:
It is not a requirement to explicitly include ABC in this class. It will behave the same with or without this declaration - as an abstract class.